nexus/posts/serializers.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework import serializers
+
+from . import models
+
+
+class AdminPostSerializer(serializers.ModelSerializer):
+    posted_by = serializers.EmailField(source='posted_by.email', read_only=True)
```

Comment:
This should be `posted_by__email` rather than `posted_by.email`

Reply:
Serializer field's source attribute takes only `.` (dotted) notations. [source](https://www.django-rest-framework.org/api-guide/fields/#source)